app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1108,7 +1111,8 @@ DeviceDelegateOpenXR::EndFrame(const FrameEndMode aEndMode) {
   };
 
   auto canAddLayers = [&layers, maxLayers = m.systemProperties.graphicsProperties.maxLayerCount]() {
-      return layers.size() < maxLayers;
+      // Some runtimes incorrectly report 0 as maxLayerCount like Spaces.
+      return layers.size() < std::max(1u, maxLayers);
```

Comment:
Wouldn't this change hide potential bugs in other runtimes ? Perhaps a DEBUG message would at least help us detect them (would an ASSERT be too much ?)

Reply:
An ASSERT will make the workaround useless :)

But yeah I guess we could add a debug message, although I'll do it outside this call because it's called many times for every frame.